packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js

```diff
@@ -34,20 +36,24 @@ const setup = (props) =>
   );
 
 describe('BlocksEditor', () => {
-  it('should render blocks without error', () => {
-    setup();
+  it('should render blocks without error', async () => {
+    setup({ value: null });
 
     expect(screen.getByText('blocks type')).toBeInTheDocument();
+    expect(screen.getByText('blocks description')).toBeInTheDocument();
+    await waitFor(() => {
+      expect(screen.getByText('blocks placeholder')).toBeInTheDocument();
+    });
```

Comment:
I'm a bit surprised that waitFor is necessary, do you know why?

Reply:
actually I struggled to locate placeholder for quite sometime, but it appears placeholder gets updated in DOM quite late.